src/fitnesse/responders/account/SaveAccountResponder.java

```diff
@@ -0,0 +1,64 @@
+package fitnesse.responders.account;
+
+import fitnesse.FitNesseContext;
+import fitnesse.authentication.Password;
+import fitnesse.http.Request;
+import fitnesse.http.Response;
+import fitnesse.http.SimpleResponse;
+import fitnesse.responders.BasicResponder;
+import fitnesse.responders.ErrorResponder;
+import org.apache.commons.lang3.StringUtils;
+
+public class SaveAccountResponder extends BasicResponder {
+  @Override
+  public Response makeResponse(FitNesseContext context, Request request) throws Exception {
+    if (request.getAuthorizationUsername() == null) {
+      return getResponse(context, "You have to be logged in to use this feature.");
+    }
+    Password password = new Password();
+    if (request.hasInput("changePassword")) {
+      String passwordText = StringUtils.trim(request.getInput("PasswordText"));
```

Comment:
Also request and validate the current password of the user before allowing it to be changed